### DISCUSSION TITLE
Freeze protection

### DISCUSSION THREAD

**Discussion by kgreene1983** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#271] [General]
Just wondering if with this modification does the freeze protection work?

**Comment by visualapproach** [comment]
The code starts the heater below 10C, so yes. Without warranty :)

**Comment by kgreene1983** [comment]
Well it's just so happens that this morning my temp was reading  9 and it hadn't come on? 

![Screenshot_2022-05-12-07-15-11-954_com android chrome](https://user-images.githubusercontent.com/32735113/167928247-71c0f36f-a007-4607-9305-61e2e03236a0.jpg)

**Reply by visualapproach** [reply]
Hm I thought you had a 4-wire. If you have a 6-wire the antifreeze is the factory setting. The cio is turning on heat around 5C and you should probably not automate a pump stop in that case.

**Reply by kgreene1983** [reply]
I'm on the 6 wire 103 model... So I should leave the pump running?

**Reply by visualapproach** [reply]
I'd think it starts automatically. Better watch it to be sure. It seems different models have a slightly different behavior. But the esp shouldn't change the behavior, what I meant was that if you have an automation to turn off the pump just after the antifreeze kicks in, you are in untested territory

**Comment by Bishbosh02** [comment]
My pump as has this feature (6 wire S100102) below XXc pump and heater turn on and run until above 10c this is a pain if the unit is on the bench for testing.

Cant confirm the XXc temp as the water does not get below 14c now but think it was 6c
